<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serialize</class>
 <widget class="QDialog" name="Serialize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>465</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Drawish</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>120</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>120</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Perform selected operations on all images in a folder.
The operations will be performed in the order in which they are listed.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonChooseFolder">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Select folder</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkGrayscale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Grayscale</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkContrast">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>88</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Contrast</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBright">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Brightness</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkCrop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkRotate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkPaste">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Paste image</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkScale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkConvert">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinContrast">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-50</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBright">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-50</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCropx">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Initial point from the left</string>
   </property>
   <property name="placeholderText">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCropy">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Initial point from the top</string>
   </property>
   <property name="placeholderText">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCropw">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Width</string>
   </property>
   <property name="placeholderText">
    <string>W</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCroph">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Height</string>
   </property>
   <property name="placeholderText">
    <string>H</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePastePath">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Drag / write path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePastey">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePastex">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineScaleh">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>New height</string>
   </property>
   <property name="placeholderText">
    <string>H</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineScalew">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>New width</string>
   </property>
   <property name="placeholderText">
    <string>W</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboSuffix">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>BMP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ICO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>JPEG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PNG</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>120</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>120</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>120</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>120</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_origin">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboRotateDirection">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>79</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Left</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Right</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboRotateOnly">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>All</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vertical images only  ▯ →▭</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Horizontal images only  ▭ →▯</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="textErrors">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioFormFactor">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>332</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Overrides W H scaling </string>
   </property>
   <property name="text">
    <string>Fit the form factor</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1:1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2:1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1:2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2:3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3:2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3:4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4:3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3:5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5:3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9:16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16:9</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
